src/Modules/LearningTasks/Tutor.LearningTasks.Core/Domain/Activites/Example.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Tutor.BuildingBlocks.Core.Domain;
+
+namespace Tutor.LearningTasks.Core.Domain.Activites;
+
+public class Example : Entity
+{
+    public string Description { get; private set; }
```

Comment:
Check out the heuristics described [here](https://github.com/psw-ftn/supportive-information/tree/master/s2/w1-w2#1-kreiranje-modela-takti%C4%8Dkih-ddd-%C5%A1ablona-za-kontekst-kartice).

The guiding question to ask is - what will I load together for the learner's use case and what will I load together for the instructor's use case.

If I do not want to load each example at the start of the activity, I should treat it as being outside the aggregate. If I do want to load it, it is inside the aggregate. If it has an explicit lifecycle it is an entity. If it does not, it is a value object.

Reply:
I don't want to load each example at the start of the activity, so I will treat it as being outside the aggregate and add `ActivityId` to example. 
I want to have a use case where the instructor can edit examples, so I think it is easier to implement if the object has an `id`. Therefore, I will leave Example as an entity.